### PR TITLE
docs(ipx): instructions for modifiers

### LINKED ISSUE #351: Animated images don't work
As an example,

```html
 <nuxt-img
        src="/team/images/image.gif"
        height="200px"
        width="200px"
      />
```
does not work. But

```html
  <nuxt-img
        src="https://media3.giphy.com/media/Jzxgefavt2aB2/giphy.gif"
        height="200px"
        width="200px"
      />
```
works fine. This is probably some problem on part of the default provider `ipx`. Is there a way to fix this?

### PR DESCRIPTION
**Pull request by IgnisDa**
Fixes #351.

Add instructions on how to use IPX modifiers and
also instructions on how to use animated images.

### CODE REVIEW

**Review by danielroe — APPROVED** [review]
Thank you ❤️

File: docs/content/en/4.providers/ipx.md [hunk 1]
```diff
@@ -7,6 +7,24 @@ navigation:
...
+
+[Modifiers](https://github.com/unjs/ipx/#modifiers) supported by IPX can be used.
+
+```html
+<nuxt-img src="/image.png" :modifiers="{ grayscale: true, tint: '#00DC82' }" />
+```
+
+### Animated images
+
+They require an additional modifier and work by converting to webp
```
Comment (pi0) [inline docs/content/en/4.providers/ipx.md:20]:
```suggestion
**Note:** Animated support is currently experimental. When used, `gif` format is converted to `webp`
```

File: docs/content/en/4.providers/ipx.md [hunk 2]
```diff
@@ -7,6 +7,24 @@ navigation:
...
+([browser support](https://caniuse.com/webp)). Resize operations are not yet
```
Comment (pi0) [inline docs/content/en/4.providers/ipx.md:21]:
```suggestion
([check browser support](https://caniuse.com/webp)). Setting size is also not supported yet
```

**Review by pi0 — APPROVED** [review]

**Comment by pi0** [comment]
Thanks @IgnisDa

**Pull request merged**